pkg/apiserver/api_installer.go

```diff
@@ -378,7 +379,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		Creater:          a.group.Creater,
 		Convertor:        a.group.Convertor,
 		Codec:            mapping.Codec,
-		APIVersion:       a.group.Version,
+		APIVersion:       apiutil.GetGroupVersion(a.group.Group, a.group.Version),
```

Comment:
Yes, that needs to be updated, and yes we need to update the parameter name in RESTMapping, and I believe there are many other places that use the name "version" but actually contains "group/version".
I prefer to do these refactoring one by one. I don't want to bloat this one too much. What do you think?